test/test-applications/integrations/TestApplication.CustomSdk/Program.cs

```diff
@@ -29,14 +32,39 @@ public static class Program
     private static readonly ActivitySource ActivitySource = new(
         "TestApplication.CustomSdk");
 
+    private static readonly Meter Meter = new("TestApplication.CustomSdk");
+    private static readonly Counter<long> Counter = Meter.CreateCounter<long>("RequestCounter");
+
     public static async Task Main(string[] args)
     {
         ConsoleHelper.WriteSplashScreen(args);
 
         // if auto-instrumentation is not injecting sdk
         // then it's client's code responsibility
         // to subscribe to all activity sources
-        using var tracerProvider = Sdk
+
+        using var tracerProvider = BuildTracerProvider();
+
+        using var meterProvider = BuildMeterProvider();
+
+        await SendNServiceBusMessage();
+
+        Counter.Add(1);
+
+        using (var activity = ActivitySource.StartActivity("Manual"))
+        {
+            await PingRedis(args);
+
+            using var client = new HttpClient();
+            await client.GetStringAsync("https://www.bing.com");
+        }
+
+        Thread.Sleep(TimeSpan.FromSeconds(10));
```

Comment:
Do we really need this sleep here? It is also quite long.

Reply:
I think it is not needed...

Reply:
Changed in https://github.com/open-telemetry/opentelemetry-dotnet-instrumentation/pull/2081/commits/817994092dd4edee8dbefedf3a41ca148abab5d2 in order for it not to be needed anymore